proposal/benchen216/README.rst

```diff
@@ -0,0 +1,64 @@
+==========================================
+Proposal for logic gate circuit simulation
+==========================================
+
+Basic information
+=================
+In recent days, FPGA SoC and some kinds of SoC can be use as AI or deep learning accessor,
+also 5g will use FPGA as RFSoc to extend the radio transmission.
+
+project URL:
+
+https://github.com/benchen216/logic-gate-sim
+
+
+Perspective users
+=================
+
+1. The student that feel verilog is so difficult.
+2. The people want to run simple logic circuit.
+
+System architecture
+===================
+
+Data Flow
+
+1. read in logic gate circuit syntax
```

Comment:
I assume the data flow can be fully controlled and implemented in Python or a scripting language.  Where would C++ be useful?